src/main/java/edu/cmu/cs/cs214/rec10/plugin/SortWithSwapsPlugin.java

```diff
@@ -0,0 +1,118 @@
+package edu.cmu.cs.cs214.rec10.plugin;
+
+import edu.cmu.cs.cs214.rec10.framework.core.GameFramework;
+import edu.cmu.cs.cs214.rec10.framework.core.GamePlugin;
+
+import java.util.Random;
+
+public class SortWithSwapsPlugin implements GamePlugin {
+    private int ONE = 1;
```

Comment:
Do these values come from somewhere? constants? Might not be optimal to hard-code them.